src/handler.rs

```diff
@@ -22,13 +22,25 @@ impl Handler {
         self.backtrace.as_ref()
     }
 
+    /// Store a captured `Backtrace` in this handler
+    pub fn set_backtrace(&mut self, backtrace: Option<Backtrace>) {
+        self.backtrace = backtrace;
+    }
```

Comment:
I'm considering whether it's better to take an `Option<Backtrace>` or a `Backtrace` here.

If it takes Option it makes it possible to drop an inner backtrace by setting it to `None`. I have trouble imagining when this would be needed. I can imagine wanting to avoid printing the backtrace in the output, but that should be possible with https://github.com/yaahc/color-eyre/pull/113 already (which I'm just realizing I may have never cut a release for, I should really get around to that 😅).

On the flip side, this makes it possible to accidentally set it to `None` as well, which seems like a potential footgun.

I think generally I'm leaning towards wanting this to take a `Backtrace` directly, and the same for the spantrace setter below as well. Let me know if there's any other particular reason I didn't consider which would make you prefer `Option`.